### DISCUSSION TITLE
Installing telegram packages to a project

### DISCUSSION THREAD

**Discussion by colintkn** [python-telegram-bot/python-telegram-bot#2754] [Q&A] [answered]
Currently, I'm thinking of making changes to the python-telegram-bot package code for my app. 

The installation as followed by the guide puts the telegram package at the following address:
`
Library > Frameworks > Python.framwork > Versions> 3.9 > lib > python3.9 > site-packages > telegram 
`

The changes to the package is only project-specific. How do I replicate this whole package into my project, such that it references the project-specific package. 

I tried copy and pasting the telegram package into my project. 

E.g
![image](https://user-images.githubusercontent.com/65516485/139487363-479234a4-baf6-4f22-bf6a-48fe4fb52d04.png)

`
Desktop > ProjectFolder > telegram
`

![image](https://user-images.githubusercontent.com/65516485/139486955-d4b4d694-ec13-4640-94cc-dc22f03aebaa.png)

However, the import still references original package at:
`
Library > Frameworks > Python.framwork > Versions> 3.9 > lib > python3.9 > site-packages > telegram 
`
How do I work around this.

**Comment by Poolitzer** [comment, marked as answer]
Hi. What stops you from installing the requirements of this project "manually" and just create your own telegram folder and git clone this repo's telegram folder into it? I would use the git submodule if you are in a git repo.

**Reply by colintkn** [reply]
Thank you. I created a virtualenv and installed the telegram packages in the venv. 

https://stackoverflow.com/questions/53925660/installing-python-dependencies-locally-in-project

For those who are following this, 
If you're using VSCode, remember to change the interpreter to the python installed in the virtual environment.
